Content.Client/_Harmony/JoinQueue/QueueState.cs

```diff
@@ -0,0 +1,54 @@
+using Content.Shared._Harmony.Common.JoinQueue;
+using Robust.Client.Audio;
+using Robust.Client.Console;
+using Robust.Client.State;
+using Robust.Client.UserInterface;
+using Robust.Shared.Player;
+
+namespace Content.Client._Harmony.JoinQueue;
+
+public sealed class QueueState : State
+{
+    [Dependency] private readonly IUserInterfaceManager _userInterface = default!;
+    [Dependency] private readonly IClientConsoleHost _console = default!;
+
+
+    private const string JoinSoundPath = "/Audio/Effects/newplayerping.ogg";
```

Comment:
Use a sound specifier instead
```suggestion
    private static readonly SoundSpecifier JoinSoundPath = new SoundPathSpecifier("/Audio/Effects/newplayerping.og");
```